app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Users::InvitationsController < Devise::InvitationsController
+  protected
+    # Override require_no_authentication method defined at DeviseController (parent of Devise::InvitationsController)
+    def require_no_authentication
+      super
+      if flash[:alert].present?
+        flash[:alert] = _('Cannot validate invitation token unless you sign out')
```

Comment:
Looks good @jollopre just update the text of the alert here to match @mxsstrong recommendation in the ticket (switch :alert to :notice): "Notice: You are already logged in as another user. Please log out to activate your invitation"